sdk/src/types/api/plugins/participation/mod.rs

```diff
@@ -6,4 +6,5 @@
 
 pub mod error;
 pub mod responses;
+#[allow(non_camel_case_types)]
```

Comment:
I couldnt find a non-camel-case, is this needed?

Reply:
It's an error in the derive serde repr crate 🤷‍♀️

Reply:
I moved it so it's more clear